docs/dev/Demo/Slice Demo Prep/Prime-Demo-Script-Postman-Setup.md

```diff
@@ -0,0 +1,103 @@
+# Utilizing the GitHub Workflow for designing a Slice Demo Postman Collection:
+<!-- Table of Contents auto-generated with `bin/generate-md-toc.sh` -->
+
+<!-- toc -->
+
+* [Slice Demo Prep Basics: Playing the Prime](#slice-demo-prep-basics-playing-the-prime)
+* [Getting Started](#getting-started)
+    * [Creating a Collection from scratch](#creating-a-collection-from-scratch)
+        * [Setting up with Index.js](#setting-up-with-indexjs)
+
+
+Regenerate with "pre-commit run -a markdown-toc"
+
+<!-- tocstop -->
+## Slice Demo Prep Basics: Playing the Prime:
+During the slice demo there will be various points of handoff from the TOO/TIO to the Prime and vice versa. When playing the prime, we recommend developing visuals that are split up into 2 sections:
+#### Handoff from TOO -> Prime
+    Visual 1: Prime receives a notification of a move. Prime updates the move details. Prime performs another action (e.g. creates a service item)
+#### Handoff from Prime --> TOO 
+After the Prime creates a Service Item, there is a verbal handoff to the TOO, nothing technical happens
+#### Handoff TOO --> Prime
+The TOO reviews the Service Item and approves them and there is a verbal handoff to the Prime to show how a payment request is created and proof od service is uploaded
+    
+    Visual 2: Prime creates a payment request and uploads proof of service.
+#### Handoff from Prime to TOO/TIO
+After the prime uploads the proof of service, they submit the payment request ID to teh TIO and do a verbal handoff to teh TOO.
+## Getting Started:
+1. Clone the `prime_api_deliverable` repo: `git@github.com:transcom/prime_api_deliverable.git`
+2. Navigate to the `postman` folder, where you will find previous slice demo collections (e.g. 202108-slice-demo)
+3. Copy over contents from a previous collection or create a new collection from scratch.
+
+### Creating a Collection from scratch:
```

Comment:
I think section is very clear! And it's helpful that you called out the expected file structure and naming conventions